main.py

```
@@ -70,6 +77,150 @@ def main():
+                        config['port']
+                    )
+                    
+                    # 保存监听器命令到文件
+                    listener_file = os.path.join(config['output_dir'], "listener_commands.txt")
+                    with open(listener_file, 'w', encoding='utf-8') as f:
+                        f.write(listener_cmd)
+                        f.write(f"\n\n# MSF生成命令:\n# {msf_command}")
+                    
+                    # 打印结果
+                    if not quiet:
+                        print(f"\n✅ MSF载荷生成成功！")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)